scoper.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Maybe we should go for https://github.com/nette/neon? Unlike JSON, multiline is either which could help making the code a bit clearer. Alternatively we could allow a PHP config.

To be clear I don't have anything against JSON and it's a popular config format, but in our case it looks really awkward to use